src/cluster.h

```diff
@@ -127,6 +127,7 @@ typedef struct clusterNode {
                                     may be NULL even if the node is a slave
                                     if we don't have the master node in our
                                     tables. */
+    unsigned long long last_in_ping_message; /* The number of the last carried in the ping message */
```

Comment:
i suppose we better zero this in createClusterNode, otherwise there's a slim chance for some bug.

Reply:
you are right, I had initialized the var in createClusterNode, please see https://github.com/judeng/redis/blob/fca6958d4b2a999e680de0a3bd1878c2e5c4cb2f/src/cluster.c#L967